src/AppBundle/Command/ImportStdCommand.php

```diff
@@ -862,6 +862,20 @@ protected function importLocationData(Card $card, $data)
 		}
 	}
 
+	protected function importEnemyLocationData(Card $card, $data)
+	{
+		$mandatoryKeys = [
+			'shroud',
+			'clues',
```

Comment:
Although all enemy_locations right now have `clues: null`, I think it makes sense that this is consistent with the `location` type.